lib/cms/repo.ex

```diff
@@ -130,7 +131,28 @@ defmodule CMS.Repo do
   def do_banner do
     case @cms_api.view("/cms/important-notices", []) do
       {:ok, [api_data | _]} -> Banner.from_api(api_data)
-      {:ok, _} -> :empty
+      {:ok, _} -> nil
+      {:error, _} -> :error
+    end
+  end
+
+  @spec photo() :: Photo.t() | nil
+  def photo do
+    cached_value = do_photo()
+
+    if cached_value == :empty || cached_value == :error, do: nil, else: cached_value
```

Comment:
This may be made obsolete depending on how you want to address [this comment](https://github.com/mbta/dotcom/pull/2695/files#r2291920948), but if the implementation of `do_photo()` is kept as it currently is, then `cached_value` can never be equal to `:empty`